tests/VolleyManagement.UnitTests/Mvc/Controllers/TournamentsControllerTests.cs

```diff
@@ -1678,24 +1676,31 @@ private Tournament MakeTestTournamentWithOneDivision(int tournamentId)
         {
             var tournament = MakeTestTournament(tournamentId);
             tournament.Divisions = CreateTestDivisions();
-            tournament.Divisions.Remove(tournament.Divisions[1]);
+            tournament.Divisions.Remove(tournament.Divisions.ElementAt(1));
             return tournament;
         }
 
         private Tournament MakeTestTournamentWithTenDivisions(int tournamentId)
         {
             var tournament = MakeTestTournament(tournamentId);
             tournament.Divisions = new List<Division>();
+            IEnumerator<Division> divEnumerator = default(IEnumerator<Division>);
```

Comment:
You learned about enumerators so now you are going to use them anywhere? )))
Rewrite line 1691:1690.

Reply:
haha, but it is the most clear and convenient way to iterate icollection,

Reply:
it does not mean it has to be used everywhere